### PR TITLE
Ebs snapshot updates

### PR DESCRIPTION
**Pull request by ocampocj**

### CODE REVIEW

File: c7n/resources/ebs.py [hunk 1]
```diff
@@ -208,7 +209,7 @@ def get_instance_mapping(self, resources):
             InstanceIds=instance_ids)
         self.log.debug("Queried %d instances for %d volumes" % (
             len(instances), len(resources)))
-        return {i['InstanceId']: i for i in instances}
+        return {i['InstanceId']: i for i in instances}
```
Comment (kapilt) [inline c7n/resources/ebs.py]: stray space

**Comment by kapilt** [comment]
lgtm, although wondering if we shouldn't just do this generically in the query meta class in query.py if service == ec2

**Comment by jeffastorey** [comment]
@kapilt agreed it would be good to have this more generically, but short term this would be pretty helpful to allow mark-delete for EBS snapshots so we don't inadvertently delete data

**Comment by kapilt** [comment]
the generic version has landed, going to close this one out.

**Comment by kapilt** [comment]
fwiw merged in #369

**Pull request closed without merging**